docs/GETTING-STARTED.md

```diff
@@ -1,43 +1,34 @@
 
-## Npm tasks desciption
+## Running the application
+
+Most tasks are wrappers for `rctmake`.
+
+#### Npm tasks desciption
 Npm tasks are run via 
 ```bash
-npm run <TASK>
+npm run <TASK> -- RCTMAKE_ADDITIONAL_ARGUMENTS
 ```
+#### Within docker:
+1. `dev` - main task for running application in dev mode, see `rctmake` help. The task prune, build and run containers, one for the database and one for the application. Changes in the local repo force the application restart (see nodemon and the `start:dev` script, see also `rctmake` option `--target-modifier`). You can run multiple instances of application in dev mode simultaneously, see `rctmake` option `--subtarget` and `--rct-http-port`.
+2. other `*dev*` tasks are modification of pervious one.
+#### Starting locally:
+ 1. `start:dev:local` - run the RCT application locally, by default it fetches env vars from `<ProjectDir>/docker/dev.env` with exception that `RCT_DB_HOST` env var is substituted to localhost. If you want to make other substitutions, specify them as follows (bash) before the aplication start: `(export FOO=bar; npm run start:dev:local)`
+ 2. `deploy:db:local` - deploy the database locally with the use of `<ProjectDir>/database/setup-db.sh` script. It uses the same env as the upper task and follows the same env vars substitution logic.
 
-### Within docker:
-1. `prune` - removes each container related to this project
-
-2. `dev` - prune, build and run containers, one for the database and one for the application. Changes in the local repo force the application restart on docker (see nodemon and the `start:dev` script).
-Dev does not deploy the new database, it only clears it. In case of any changes in the database or docker images, the use of `dev:prune` is advised.
-3. `dev:up` - up stopped (previously) built containers (no data in db is erased)
-4. `dev:up-r <DUMP_NAME>`  - as dev:up but also restore specified dump (existing in `<ProjectDir>/database/cache/dumps`), see the `dump:list` task
-5. `app:attach` - attach to the application container console
-6. `db:check` - export the current pgmodeler design as the database definition file, delete the database container and rebuild it with the new design. Application container will be deleted as well.
-7. `db:attach` - run psql as postgres (connected to RCT database) on database container
-8. `db:clean` - clean database in the running database docker
-
-### Database dumps managing, dumps are stored in `<ProjectDir>/database/cache/dumps`
-9. `dump:make <DUMP_NAME>` - make a data dump and store it under the specified name
-10. `dump:list` - list all the dumps stored in local cache
-11. `dump:restore <DUMP_NAME>` - clean the db and restore the data from the specified dump file
-12. `dump:remove <DUMP_NAME>` - remove the dump file from cache
-
-### Testing
+#### Testing
+15. `test` - main task for testing purpose, run static analysis and codecov on docker containers, results are available in `<ProjectDir>/reports/`
```

Comment:
If you want to have a numbered list here, please at least make sure that the numbers are in order.

Reply:
Done